client/beefy/src/worker.rs

```diff
@@ -262,8 +262,10 @@ where
 					return
 				};
 
+			let mut payload = Payload::default();
+			payload.push(known_payload_ids::MMR_ROOT_ID, mmr_root);
```

Comment:
nit: we could have `Payload::new` to simplify this and avoid marking `payload` as `mut`.

Reply:
like 

```rust
let payload = Payload::new().push(known_payload_ids::MMR_ROOT_ID, mmr_root);
```

technically that requires modifying the `push` method to take `self` and return it so we can chain multiple calls to `push`

Reply:
Yeah, or just `Payload::new(known_payload_ids::MMR_ROOT_ID, mmr_root);` - that additionally ensure that no empty payload can be created.